### PR TITLE
add folder nav buttons

### PR DESCRIPTION
**Pull request by roundy7722**
once a NAM/IR is selected, nav buttons (up/down) will appear to quickly load the files in the selected NAM/IR's containing folder.
if the NAM/IR is cleared, the nav buttons will become hidden, but will re-appear upon new selection.
if the NAM/IR selected is the only file in the folder, the buttons will not appear.

this is currently only in the NeuralAmpModeler-app project.  if you'd like to move forward with this we can get the change in the other projects too.

Capture

### CODE REVIEW

**Comment by sdatkinson** [comment]
Really excited to have a look at this! But it'll probably be a day before I have time. Hang tight!

**Comment by bnkwsk** [comment]
@roundy7722 it would be much easier to review if the `NeuralAmpModeler/NeuralAmpModeler.cpp` file formatting would be left as is. If we need a reformat, let’s have it in a separate PR. Could you revert formatting changes? 🙏🏼

**Comment by olilarkin** [comment]
There is an iPlug2 control already that can help with this, the IVDiskPresetManagerControl. 

```
pGraphics->AttachControl(new IVDiskPresetManagerControl(pGraphics->GetBounds().GetFromTop(50), "/Users/oli/Downloads/NAM_models-main", "nam"));
```

**Comment by roundy7722** [comment]
> @roundy7722 it would be much easier to review if the `NeuralAmpModeler/NeuralAmpModeler.cpp` file formatting would be left as is. If we need a reformat, let’s have it in a separate PR. Could you revert formatting changes? 🙏🏼

ugh - i see that now looking at the diff.  will try to re-commit with no reformatting.

**Comment by sdatkinson** [comment]
There's `format.bash` which should be really helpful :)

File: NeuralAmpModeler/NeuralAmpModeler.cpp [hunk 1]
```diff
@@ -15,44 +16,50 @@
 
 using namespace iplug;
 using namespace igraphics;
+namespace fs = std::filesystem;
```
Comment (sdatkinson) [inline NeuralAmpModeler/NeuralAmpModeler.cpp:19]: Oh dear...there are _so many_ changes in this due to formatting.

I need this to be formatted correctly before I can review.

**Review by sdatkinson — CHANGES_REQUESTED** [review]
Please run [`format.bash`](https://github.com/sdatkinson/NeuralAmpModelerPlugin/blob/main/format.bash)

**Comment by sdatkinson** [comment]
> There is an iPlug2 control already that can help with this, the IVDiskPresetManagerControl.
> 
> ```
> pGraphics->AttachControl(new IVDiskPresetManagerControl(pGraphics->GetBounds().GetFromTop(50), "/Users/oli/Downloads/NAM_models-main", "nam"));
> ```
> 
> see also `IDirBrowseControlBase`

I think I'm keen on seeing this solution. Sounds like it may be both simpler and get us making use of the iPlug2 framework better.

File: NeuralAmpModeler/resources/main.rc [hunk 2]
```diff
@@ -239,3 +239,6 @@ HELP_FN SVG HELP_FN
 /////////////////////////////////////////////////////////////////////////////
 #endif    // not APSTUDIO_INVOKED
 
+DOWN_ARROW_FN SVG DOWN_ARROW_FN
```
Comment (sdatkinson) [inline NeuralAmpModeler/resources/main.rc:242]: Comment about the arrow graphics: can the insides be white? Otherwise they're hard to see.

Comment (ServusJon) [reply NeuralAmpModeler/resources/main.rc:242]: yes, please make it more usable :)

File: NeuralAmpModeler/NeuralAmpModeler.cpp (hunk 1 above)
Comment (roundy7722) [reply NeuralAmpModeler/NeuralAmpModeler.cpp:19]: new commit with fixed formatting

**Pull request closed without merging**